bbffb7acdcda268a2b1b7c33c89c4abc2f93ec87 breaks compile

@TurboGit element got declared as a constant in _action_process_rating, but it gets assigned to in the function

EDIT: 

```
/home/bill/src/darktable/darktable/src/common/ratings.c: In function ‘_action_process_rating’:
/home/bill/src/darktable/darktable/src/common/ratings.c:275:17: error: assignment of read-only parameter ‘element’
  275 |         element = DT_RATINGS_UPGRADE;
      |                 ^
compilation terminated due to -Wfatal-errors.

```

--- a/src/common/ratings.c
+++ b/src/common/ratings.c
@@ -256,7 +256,7 @@ enum
 };
 
 static float _action_process_rating(gpointer target,
-                                    const dt_action_element_t element,
+                                    dt_action_element_t element,
                                     const dt_action_effect_t effect,
                                     const float move_size)
 {

